R/paginate.R

```diff
@@ -108,6 +112,14 @@ req_paginate <- function(req,
 
 #' Perform a paginated request
 #'
+#' @description
+#' `r lifecycle::badge("experimental")`
+#'
+#' * `paginate_req_perform()` requests all pages for a paginated request and
+#'   returns a list of response.
+#' * `paginate_next_request()` generates the next request for a paginated
```

Comment:
Maybe something like `paginate_next_request()` generates the request to the next page